tests/unit/abilities-api/wpRegisterAbility.php

```diff
@@ -315,13 +315,13 @@ public function test_permission_callback_receives_input(): void {
 		do_action( 'abilities_api_init' );
 
 		$received_input                                       = null;
```

Comment:
CI on WP core will complain, need to fix spacing here and most likely in a few other places.